tests/docker/ducker-ak

```diff
@@ -217,10 +221,40 @@ $((${duration} % 60))s.  See ${build_log} for details."
 $((${duration} % 60))s.  See ${build_log} for details."
 }
 
+docker_run() {
+    local node=${1}
+    local image_name=${2}
+
+    # Invoke docker-run. We need privileged mode to be able to run iptables
+    # and mount FUSE filesystems inside the container.  We also need it to
+    # run iptables inside the container.
+    must_do 1 0 docker run --privileged \
+        -d -t --net=host -h "${node}" --network ducknet \
+        --memory=${docker_run_memory_limit} --memory-swappiness=1 \
+        -v "${kafka_dir}:/opt/kafka-dev" --name "${node}" -- "${image_name}"
+}
+
+setup_custom_ducktape() {
+    local custom_ducktape="${1}"
+    local image_name="${2}"
+
+    [[ -f "${custom_ducktape}/README.md" ]] || \
+        die "You must supply a valid ducktape directory to --custom-ducktape"
+    docker_run ducker01 "${image_name}"
+    local running_container="$(docker ps -f=network=ducknet -q)"
+    must_do 1 1 docker cp "${custom_ducktape}" "${running_container}:/opt/ducktape"
+    docker exec --user=root ducker01 bash -c 'set -x && cd /opt/ducktape && python ./setup.py develop && cd /opt/kafka-dev/tests && python ./setup.py develop'
+    [[ $? -ne 0 ]] && die "failed to install the new ducktape."
+    must_do 1 1 docker commit ducker01 "${image_name}"
+    must_do 1 0 docker kill "${running_container}"
+    must_do 1 0 docker rm ducker01
```

Comment:
Could we not use something a bit more descriptive for the must_do parameters? It's a little hard to read at the moment.